fastn/django/__init__.py

```diff
@@ -17,6 +18,10 @@
 CI = utils.AESCipher(SECRET_KEY)
 COOKIE_MAX_AGE = 365 * 24 * 60 * 60
 
+FASTN_AUTH_CALLBACK = getattr(
+    settings, "FASTN_AUTH_CALLBACK", "fastn.django.default_callback"
+)
```

Comment:
You can do the `module_loading.import_string` resolution part here as well to only do it once, instead on per request.

Reply:
[done](https://github.com/fastn-stack/fastn_py/pull/1/commits/1e414934204bf04152314daf3dcc7e7c811a3ce6)